tools/mo/openvino/__init__.py

```diff
@@ -1,4 +1,72 @@
+# -*- coding: utf-8 -*-
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+try:
+    # Internal modules
+    from openvino import test_utils as test_utils
+except ImportError:
+    pass
+
+try:
+    # Import all public modules
+    from openvino import runtime as runtime
+    from openvino import frontend as frontend
+    from openvino import helpers as helpers
+    from openvino import preprocess as preprocess
+    from openvino import utils as utils
+    from openvino.runtime import properties as properties
+    # Import old API
+    # TODO: remove in 2024.0
+    from openvino import inference_engine as inference_engine
```

Comment:
I suggest also moving this import to a dedicated try / catch, because user can miss installation of API 1.0 requirements and it will not be build. 
So, for each group of functionality, let's have a dedicated block

Reply:
Agree, will push with next commit